identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery-confirm.jsp

```diff
@@ -54,15 +55,28 @@
 
     String confirmationKey = request.getParameter("confirmation");
     String callback = request.getParameter("callback");
```

Comment:
Let's update this variable name to callbackUrl as well, for enhanced readability, if possible :)